framework/OutStreams/OutStreamPlot.py

```diff
@@ -331,23 +331,24 @@ def __fillCoordinatesFromSource(self):
             outputIndexes = self.sourceData[pltIndex].indexes if xSplit[1].lower() == 'output' else []
             if xSplit[2].strip() not in self.sourceData[pltIndex].getVars(xSplit[1].lower())+outputIndexes:
               self.raiseAnError(IOError, 'Not found variable "'+ xSplit[2] + '" in "'+xSplit[1]+ '" of DataObject "'+ self.sourceData[pltIndex].name+'"!')
-            self.xValues[pltIndex][cnt].append(dataSet.isel(False,RAVEN_sample_ID=cnt)[xSplit[2]].values.astype(float, copy=False))
+            # for variable from input space, it will return array(float), not 1d array
+            self.xValues[pltIndex][cnt].append(np.atleast_1d(dataSet.isel(False,RAVEN_sample_ID=cnt)[xSplit[2]].values.astype(float, copy=False)))
```

Comment:
does this affects other plots in the tests?

Reply:
As far as I know, this will not affects other plots, without this change, the plots for data mining will fail.